paperless-ng/redis.conf

```diff
@@ -300,7 +300,7 @@ always-show-logo yes
 #
 #   It is also possible to remove all the previously configured save
 #   points by adding a save directive with a single empty string argument
-#   like in the following example:
+#   like in the following example:l
```

Comment:
@lociii Is this a mistake?

Reply:
Once you remove this, I can merge the PR. Everything else looks good.

Reply:
Thanks for pointing this out.
I'm on holiday and away from home this week and will only be able to fix it next week.

Reply:
Done.